src/style.css

```diff
@@ -0,0 +1,134 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  margin: 0 auto;
+  width: 100%;
+  margin-top: 10px;
+  max-width: 600px;
+  display: flex;
+  flex-direction: column;
+  box-shadow: 3px 3px 5px #494949;
+  background-color: #fbf8f8;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  height: 3%;
+  padding: 1px 10px;
+  border-bottom: 1px solid #878;
+}
+
+.todo_main {
+  display: flex;
+  flex-direction: column;
+}
+
+.todo_main form input {
+  width: 99.3%;
+  height: 40px;
+  border: none;
+  border-bottom: 1px solid #878;
+}
+
+.todo_main form input:focus-visible {
+  outline: none;
+}
```

Comment:
it needs margin-left little bit on your task input element.
![Screenshot from 2023-05-09 09-46-10](https://user-images.githubusercontent.com/86291500/237016327-5e5557a1-5e2c-4a9d-8567-13fca6c24aa0.png)